paint/elections-phragmen/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }
 sr-primitives = { path = "../../primitives/sr-primitives", default-features = false }
 phragmen = { package = "substrate-phragmen", path = "../../primitives/phragmen", default-features = false }
-paint-support = { path = "../support", default-features = false }
+support = { package = "paint-support", path = "../support", default-features = false }
```

Comment:
as soon as names are settled, one should make a PR and unify all of such imports.

Reply:
I thought we no longer want to do renames?

Reply:
indeed, but see my note: 

> as soon as names are settled, one should make a PR and unify all of such imports.

at least now that almost everywhere we use `paint-support` as `support`, having consistency is better IMO + makes `sed`-ing much easier. As mentioned, I'm more than happy to kill all of these as soon as we are done with paint -> palette.